src/euphorie/client/model.py

```diff
@@ -651,6 +652,22 @@ class ISurveySession(Interface):
     """Marker interface for a SurveySession object."""
 
 
+consultancy = Table(
```

Comment:
Why do you declare the Table in this fancy format?

Reply:
To avoid modifiying the SurveySession and Account tables themselves. If we need to remove the consultancy stuff for some reason then we can simply drop this table and don't have to fiddle with the original ones.

It follows this pattern: https://docs.sqlalchemy.org/en/13/orm/basic_relationships.html#many-to-many

... which is under “Basic Relationship Patterns”, so it's not fancy but basic. :wink: